jobs/dynatrace-oneagent/templates/pre-start.erb

```diff
@@ -121,6 +122,15 @@ setHostProps() {
     echo -n "${HOST_PROPS}" > "${hostPropsFile}"
 }
 
+createBoshDeploymentConfigFile() {
+    installLog "Creating Boshdeployment.conf file at '${BOSH_DEPLOYMENT_CONFIG_FILE}'"
+    if [ ! -f "$BOSH_DEPLOYMENT_CONFIG_FILE" ]; then
+        touch "$BOSH_DEPLOYMENT_CONFIG_FILE"
+    fi
+        echo "IsVersioned=false" > "$BOSH_DEPLOYMENT_CONFIG_FILE"
```

Comment:
Identation needs fixing here as well. It looks like the last two statements fell out of the if above.